.gitignore

```diff
@@ -126,4 +126,4 @@ dmypy.json
 .*.swp
 .idea/
 /config.yaml
-cookies.pickle
+cookies.txt
```

Comment:
This is just a tad pedantic but git ignores are very, very commonly incorrect. This should be absolute path, i.e. `/cookies.txt`, otherwise all `cookies.txt` files will be git ignored from all subdirectories.

Reply:
Happy to update this to `/cookies.txt`.